src/app/games/siam/tests/siam.component.spec.ts

```diff
@@ -57,6 +58,15 @@ describe('SiamComponent', () => {
         // Then it should fail
         await testUtils.expectClickFailure('#remainingPieces_1', RulesFailure.MUST_CHOOSE_PLAYER_PIECE());
     }));
+    it('should deselect piece when clicking a second time on the remaining pieces', fakeAsync(async() => {
+        // Given a component where the player has selected a piece for insertion
+        await testUtils.expectClickSuccess('#remainingPieces_0');
+        testUtils.expectElementToHaveClass('#remainingPieces_0_4', 'selected-stroke');
```

Comment:
ce premier expect aurait vraiment ça place dans un "it should select piece when clicking first time"

On en a deux trois (dizaines), mais les expect dans le given, c'est pas fou :p

Reply:
mais alleeeez dans un ticket précédent t'avais le discours exactement opposé : mettre un expect dans le given pour s'assurer que dans le then ça change; ici si on met jamais 'selecte-stroke', sans ce expect le test passe parfaitement alors que le comportement est pas bon.

Après, ok pour un test supplémentaire pour la première partie, je l'ajoute

Reply:
Bah, je crois qu'avant on avait pas l'expect du premier clic, du coup ça avait du sens, maintenant plus du coup oui, si y a un expect dans le given faut faire un test en plus (surtout que ça évite potentiellement de dupliquer les expect dans les givens similaires.

Reply:
ok, donc jpr ou tpr? (le test supplémentaire est là)